Content.Shared/Puppet/SharedVentriloquistPuppetSystem.cs

```diff
@@ -15,7 +16,7 @@ public override void Initialize()
     {
         base.Initialize();
         SubscribeLocalEvent<VentriloquistPuppetComponent, UseAttemptEvent>(Cancel);
-        SubscribeLocalEvent<VentriloquistPuppetComponent, InteractionAttemptEvent>(Cancel);
+        SubscribeLocalEvent<VentriloquistPuppetComponent, InteractionAttemptEvent>(CancelInteract);
         SubscribeLocalEvent<VentriloquistPuppetComponent, DropAttemptEvent>(Cancel);
```

Comment:
Why did you remind me this code exists.

Reply:
At least its not called "PuppetDummy" anymore. But the code is still quite cursed, though maybe that's appropriate for sentient ventriloquist dolls.